### ISSUE TITLE
.Or does not work together with .In

### ISSUE THREAD

**Issue by fnkr** [go-xorm/xorm#342]
```sql
  x.Cols("`repository`.*").
>   In("`team_repo`.team_id", teamIDs).
>   Or("`repository`.is_private", false).
    Join("INNER", "`team_repo`", "`team_repo`.repo_id=`repository`.id").
    GroupBy("`repository`.id").
    Find(&Repos); err != nil {
```

produces a SQL query like

```sql
SELECT `repository`.* FROM `repository` INNER JOIN `team_repo` ON `team_repo`.repo_id=`repository`.id WHERE (`repository`.is_private=?) AND ((`team_repo`.`team_id` IN (?,?))) GROUP BY `repository`.id [args] [false 2 3]
```

The `AND` should be `OR`, right? I think it's a bug :-)

**Comment by lunny** [comment]
I have fixed this issue on branch builder and it will be merged to master some time later.

**Comment by lunny** [comment]
This issue should have been resolved via v0.6.0. You can try this version. I will close this issue, and you can reopen it if you have further problem.
